generators/languages/templates/src/main/webapp/i18n/th/metrics.json

```diff
@@ -0,0 +1,101 @@
+{
+    "metrics": {
```

Comment:
@deepu105  I intended to leave it as most of them are technical term which we normally use in English.